src/imageuploadprogress.js

```diff
@@ -73,7 +73,7 @@ export default class ImageUploadProgress extends Plugin {
 
 		// Show placeholder with infinite progress bar on the top while image is read from disk.
 		if ( status == 'reading' ) {
-			viewFigure.addClass( 'ck-appear', 'ck-infinite-progress' );
+			viewFigure.addClass( 'ck-appear', 'ck-infinite-progress', 'ck-placeholder' );
```

Comment:
Why adding this class?

Reply:
I can see now how this class is used, but I got confused because we already use `ck-placeholder` but in a different case.

I think that what's wrong here is that those class names don't use BEM convention. E.g. this is image placeholder so it should contain some "image" part. Am I right, @oleq?

Reply:
`ck-image -placeholder` would be fine?

Reply:
This class is added to make sure that placeholder is always 100% width when is visible.

Reply:
It should be `ck-image-upload-placeholder` (if there's no `ck-image-upload` parent element).

What does `ck-infinite-progress` do? It should rather be `ck-image-upload-progress_infinite` (as opposed to hypothetical `ck-image-upload-progress_finite`). Or, if the "progress" is the name of the modifier, then `ck-image-upload_progress_infinite`.

`ck-appear` doesn't feel good either. It also belongs to the image upload ecosystem, so it should be at least `ck-image-upload_appear`.

Reply:
> I can see now how this class is used, but I got confused because we already use ck-placeholder but in a different case.

I agree that this class should be named better. I totally forgot about it's other usage.

Reply:
I will use `ck-image-upload-placeholder` as @oleq recommends it. I will create separate ticket to clear the class naming mess.